tools/broken-link-checker/config/ignored_targets.json

```diff
@@ -48,5 +48,6 @@
   "https://twitter.com/thekonginc",
   "https://your_okta_domain/",
   "https://github.com/Kong/kong-ee/edit/*",
-  "https://linux.die.net"
+  "https://linux.die.net",
+  "https://hub.docker.com/"
```

Comment:
```suggestion
  "https://hub.docker.com/*"
```

Reply:
@mheap do we need the `*`? I thought that the checker  would pick up the domain and exclude everything

Reply:
I recall it being an glob match (see line 50 for an example). I haven't tested though

Reply:
I tested it locally without the `*` and it worked 🤷

Reply:
The [docs](https://github.com/stevenvachon/broken-link-checker#excludedkeywords) aren't particularly clear though, but I guess it will match anything that starts with any of its values